src/matter/cluster/BasicInformationCluster.ts

```diff
@@ -28,29 +28,36 @@ const CapabilityMinimaT = ObjectT({
  * This cluster provides attributes and events for determining basic information about Nodes, which supports both
  * commissioning and operational determination of Node characteristics, such as Vendor ID, Product ID and serial number,
  * which apply to the whole Node. Also allows setting user device information such as location.
- * 
+ *
  * @see {@link MatterCoreSpecificationV1_0} § 11.1
  */
 export const BasicInformationCluster = Cluster({
     id: 0x28,
     name: "Basic Information",
 
+    /* TODO We need to add the global Attributes/Elements, see Core Spec 7.13 - with new structure maybe like that?
+            And yes some attributes could be automatically generated from the registered command implementations or such.
+    globalClusterAttributes: {
```

Comment:
What do you think of that? I could already prepare this like that when updating the others ... name is tbd, but I think with the new stricture it would be the "way to go" to do it that way :-)

Reply:
**global attributes**: I think we need to add "revision" part of the cluster definition, all other global attributes are generated from the data already specified here so don't need to be declared.

Add part of my TODO list of things to do. I would remove from this PR since it won't be done this week and won't require those declarations

Reply:
Ok, Then I move it into a COmment above - easier to convert later when the "infos" are already collected

Reply:
moved to docs of the cluster in the file so that info is not lost